CHANGELOG.md

```diff
@@ -3,8 +3,8 @@
 
 ## unreleased
 * PaymentButtons
-  * Deprecate `PaymentButtonColor` `.black`, `.silver`, `.blue`, and `.darkBlue`
-  * Add `PaymentButtonColor.gold` for `PayPalCreditButton`
+  * Undeprecate `PaymentButtonColor` `.black`, `.silver`, `.blue`, and `.darkBlue`
```

Comment:
I don't think iOS released these changes, so you are fine not having anything about button colors in the changelog.

Reply:
Oh right, the major version hasn't been published yet